client/src/app/login/login.component.html

```diff
@@ -1,11 +1,25 @@
-<div>
-    <label for="username">Username:</label>
-    <input type="text" name="username" [(ngModel)]="username">
+<div class="login-form-container" >
+
+      <img src="assets/logo2.png" alt="The big Decimal" width="180" height= "60" class="image-border">
```

Comment:
when you hardcode height and width, the proportions of the picture are not correct. When you change one, the other corrects automatically. so delete one of them. And in general it is better to define these in css, not here